src/forms/DataDrivenForm/componentMapper.js

```diff
@@ -0,0 +1,20 @@
+import { TextField, TextareaField, SwitchField } from '../FormFields/formFields';
+
+const typeMapper = (type = 'string') => ({
+  string: { component: TextField, type: 'text' },
+  integer: { component: TextField, type: 'number' },
+  boolean: { component: SwitchField },
+})[type];
+
+const widgetMapper = (type, component) => ({
```

Comment:
I love this typeMapper! Can the widget mapper and type mapper be merged together?